src/kOS/Suffixed/VectorRenderer.cs

```diff
@@ -74,7 +75,13 @@ public void ScopeLost()
             // (Note that if I didn't do this,
             // then as far as C# thinks, I wouldn't be orphaned because
             // UpdateHandler is holding a reference to me.)
-            SetShow(false);
+
+            // Store the information about lost scope to be checked in the KOSUpdate method
+            // because Unity complains if we try to set visual element visibility from
+            // outside of the main thread.  That isn't a problem most of the time, but the
+            // finalizer method called by the GC (see Variable.cs ~Variable method) is
+            // apparently called from another thread, or at least Unity thinks it is.
+            scopeLost = true;
```

Comment:
Is this duplicate information given that scopelost should be true if LinkCount <= 0?  Can we just check for that?  (and I'm not a fan of having the same spelling ``ScopeLost`` and ``scopeLost`` being things differing in time (present tense "scope is being lost right now" versus past tense "scope has already been lost").  If we keep this, can we call this ``scopeWasLost`` or something like that  (It remembers the fact that ``ScopeLost()`` has happened in the past, not right now.).